concordium-consensus/src/Concordium/ImportExport.hs

```diff
@@ -39,10 +39,18 @@
 module Concordium.ImportExport where
 
 import Control.Monad
+import Control.Monad.Catch
+import Control.Monad.IO.Class
+import Control.Monad.State (MonadState, evalStateT)
 import Control.Monad.Trans.Except
+import qualified Data.Attoparsec.Text as AP
+import Data.Bits
 import qualified Data.ByteString as BS
-import Data.List.Split
+import Data.Char (isHexDigit)
+import Data.Maybe (fromMaybe)
+import Data.Sequence
```

Comment:
I recommend importing qualified, or giving an import list of `Sequence`, since there are many names that collide with `Prelude`.